### PR TITLE
Updated Security Center API documentation links and Developer Portal links.

### PR DESCRIPTION
**Pull request by lanz**
# Description

Updated the Security Center API documentation links and the Tenable Developer Portal links.

These updates were needed due changes to the URL structure for the Security Center documentation on docs.tenable.com for product rebranding and other updates. Additionally, the links to the Developer Portal were using outdated paths.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

I verified the links would work based on my access to the docs.tenable.com and developer.tenable.com source directory structures. I also manually tested the links.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tenable/sc/policies.py [hunk 1]
```diff
@@ -284,7 +284,7 @@ def edit(self, id, **kw):
         '''
         Edits an existing scan policy
 
-        :sc-api:`scan-policy: edit <Scan-Policy.html#policy_id_PATCH>`
+        :sc-api:`scan-policy: edit <Scan-Policy.htm#policy_id_PATCH>`
```
Comment (aseemsavio) [inline tenable/sc/policies.py:287]: fragment identifier should be "policy_uuid_PATCH"

File: tenable/sc/policies.py [hunk 2]
```diff
@@ -343,7 +343,7 @@ def delete(self, id):
         '''
         Removes a configured scan policy.
 
-        :sc-api:`scan-policy: delete <Scan-Policy.html#policy_id_DELETE>`
+        :sc-api:`scan-policy: delete <Scan-Policy.htm#policy_id_DELETE>`
```
Comment (aseemsavio) [inline tenable/sc/policies.py:346]: fragment identifier should be "policy_uuid_DELETE"

File: tenable/sc/repositories.py [hunk 3]
```diff
@@ -362,7 +362,7 @@ def delete(self, repository_id):
         '''
         Remove the specified repository from Tenable.sc
 
-        :sc-api:`repository: delete <Repository.html#repository_id_DELETE>`
+        :sc-api:`repository: delete <Repository.htm#repository_id_DELETE>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:365]: fragment identifier should be "repository_uuid_DELETE"

File: tenable/sc/repositories.py [hunk 4]
```diff
@@ -381,7 +381,7 @@ def edit(self, repository_id, **kwargs):
         '''
         Updates an existing repository
 
-        :sc-api:`repository: edit <Repository.html#repository_id_PATCH>`
+        :sc-api:`repository: edit <Repository.htm#repository_id_PATCH>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:384]: fragment identifier should be "repository_uuid_PATCH"

File: tenable/sc/repositories.py [hunk 5]
```diff
@@ -473,7 +473,7 @@ def recast_risk_rules(self, repository_id, **kwargs):
         repository.
 
         :sc-api:`repository: recast rules
-        <Repository.html#RepositoryRESTReference-/repository/{repository_id}/recastRiskRule>`
+        <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/recastRiskRule>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:476]: fragment id should be "repository_uuid_recastRiskRule_GET"

File: tenable/sc/repositories.py [hunk 6]
```diff
@@ -498,7 +498,7 @@ def asset_intersections(self, repository_id, uuid=None, ip_address=None, dns=Non
         Retrieves the asset lists that a UUID, DNS address, or IP exists in.
 
         :sc-api:`repository: asst intersections
-        <Repository.html#RepositoryRESTReference-/repository/{repository_id}/assetIntersections>`
+        <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/assetIntersections>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:501]: fragment id should be "repository_uuid_assetIntersections_GET"

File: tenable/sc/repositories.py [hunk 7]
```diff
@@ -529,7 +529,7 @@ def import_repository(self, repository_id, fobj):
         '''
         Imports the repository archive for an offline repository.
 
-        :sc-api:`repository: import <Repository.html#RepositoryRESTReference-/repository/{repository_id}/import>`
+        :sc-api:`repository: import <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/import>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:532]: fragment ID should be "repository_uuid_import_POST"

File: tenable/sc/repositories.py [hunk 8]
```diff
@@ -554,7 +554,7 @@ def export_repository(self, repository_id, fobj):
         Exports the repository and writes the archive tarball into the file
         object passed.
 
-        :sc-api:`repository: export <Repository.html#RepositoryRESTReference-/repository/{repository_id}/export>`
+        :sc-api:`repository: export <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/export>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:557]: fragment ID should be "repository_uuid_export_GET"

File: tenable/sc/repositories.py [hunk 9]
```diff
@@ -585,7 +585,7 @@ def remote_sync(self, repository_id):
         Initiates a remote synchronization with a downstream Tenable.sc
         instance.  This action can only be performed on an offline repository.
 
-        :sc-api:`repository: sync <Repository.html#RepositoryRESTReference-/repository/{repository_id}/sync>`
+        :sc-api:`repository: sync <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/sync>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:588]: fragment id should be "repository_uuid_sync_POST"

File: tenable/sc/repositories.py [hunk 10]
```diff
@@ -606,7 +606,7 @@ def mobile_sync(self, repository_id):
         mobile repository specified.
 
         :sc-api:`repository: update mobile data
-        <Repository.html#RepositoryRESTReference-/repository/{repository_id}/updateMobileData>`
+        <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/updateMobileData>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:609]: fragment ID should be "repository_uuid_updateMobileData_POST"

File: tenable/sc/repositories.py [hunk 11]
```diff
@@ -627,9 +627,9 @@ def device_info(self, repository_id, dns=None, ip_address=None, uuid=None, field
         associated repository.
 
         :sc-api:`repository: device info
-        <Repository.html#RepositoryRESTReference-/repository/{repository_id}/deviceInfo>`
+        <Repository.htm#RepositoryRESTReference-/repository/{repository_id}/deviceInfo>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:630]: fragment ID should be "repository_uuid_deviceInfo_GET"

File: tenable/sc/repositories.py [hunk 12]
```diff
@@ -627,9 +627,9 @@ def device_info(self, repository_id, dns=None, ip_address=None, uuid=None, field
...
 
-        `repository: ip info <Repository.html#RepositoryRESTReference-/repository/{id}/ipInfo>`
+        `repository: ip info <Repository.htm#RepositoryRESTReference-/repository/{id}/ipInfo>`
```
Comment (aseemsavio) [inline tenable/sc/repositories.py:632]: Add ":sc-api:" in the starting of the string.

File: tenable/sc/scan_instances.py [hunk 13]
```diff
@@ -278,7 +278,7 @@ def pause(self, id):
         Pauses a running scan instance.  Note that this will not impact agent
         scan instances.
 
-        "sc-api:`scan-result: pause <Scan-Result.html#ScanResultRESTReference-/scanResult/{id}/pause>`
+        "sc-api:`scan-result: pause <Scan-Result.htm#ScanResultRESTReference-/scanResult/{id}/pause>`
```
Comment (aseemsavio) [inline tenable/sc/scan_instances.py:281]: Remove the " symbol at the start of the line.

File: tenable/sc/scan_zones.py [hunk 14]
```diff
@@ -100,7 +100,7 @@ def edit(self, id, **kw):
         '''
         Edits a scan zone.
 
-        :sc-api:`scan-zone: edit <Scan-Zone.html#zone_id_PATCH>`
+        :sc-api:`scan-zone: edit <Scan-Zone.htm#zone_id_PATCH>`
```
Comment (aseemsavio) [inline tenable/sc/scan_zones.py:103]: fragment ID should be "zone_uuid_PATCH"

File: tenable/sc/scan_zones.py [hunk 15]
```diff
@@ -154,7 +154,7 @@ def delete(self, id):
         '''
         Removes the specified scan zone.
 
-        :sc-api:`scan-zone: delete <Scan-Zone.html#zone_id_DELETE>`
+        :sc-api:`scan-zone: delete <Scan-Zone.htm#zone_id_DELETE>`
```
Comment (aseemsavio) [inline tenable/sc/scan_zones.py:157]: fragment ID should be "zone_uuid_DELETE"

File: tenable/sc/scanners.py [hunk 16]
```diff
@@ -167,7 +167,7 @@ def details(self, id, fields=None):
         '''
         Returns the details for a specific scanner.
 
-        :sc-api:`scanner: details <Scanner.html#scanner_POST>`
+        :sc-api:`scanner: details <Scanner.htm#scanner_POST>`
```
Comment (aseemsavio) [inline tenable/sc/scanners.py:170]: fragment id should be "ScannerRESTReference-/scanner"

File: tenable/sc/scans.py [hunk 17]
```diff
@@ -178,7 +178,7 @@ def list(self, fields=None):
         '''
         Retrieves the list of scan definitions.
 
-        :sc-api:scan: list <Scan.html#scan_GET>`
+        :sc-api:scan: list <Scan.htm#scan_GET>`
```
Comment (aseemsavio) [inline tenable/sc/scans.py:181]:
```
Add the ` symbol before "scan:"
```

File: tenable/sc/scans.py [hunk 18]
```diff
@@ -287,7 +287,7 @@ def details(self, id, fields=None):
         '''
         Returns the details for a specific scan.
 
-        :sc-api:`scan: details <Scan.html#ScanRESTReference-/scan/{id}>`
+        :sc-api:`scan: details <Scan.htm#ScanRESTReference-/scan/{id}>`
```
Comment (aseemsavio) [inline tenable/sc/scans.py:290]: The SC documentation does not have a way to copy the CSS fragment Identifier for some of the API endpoints.. Probably need to to reach docs team to fix it. Here's the link in question: https://docs.tenable.com/security-center/api/Scan.htm#ScanRESTReference-/scan/{id}/launch

File: tenable/sc/system.py [hunk 19]
```diff
@@ -24,7 +24,7 @@ def details(self):
         information within this call already happens upon instantiation, there
         should be little need to call this manually.
 
-        :sc-api:'system: get <System.html#system_GET>`
+        :sc-api:'system: get <System.htm#system_GET>`
```
Comment (aseemsavio) [inline tenable/sc/system.py:27]:
```
:sc-api:'system:

Here, the ' sign after "api" should be `
```

File: tenable/sc/users.py [hunk 20]
```diff
@@ -260,7 +260,7 @@ def edit(self, id, **kw):
         '''
         Edits a user.
 
-        :sc-api:`user: edit <User.html#user_id_PATCH>`
+        :sc-api:`user: edit <User.htm#user_id_PATCH>`
```
Comment (aseemsavio) [inline tenable/sc/users.py:263]: fragment ID should be "user_uuid_PATCH"

File: tenable/sc/users.py [hunk 21]
```diff
@@ -352,7 +352,7 @@ def delete(self, id):
         '''
         Removes a user.
 
-        :sc-api:`user: delete <User.html#user_id_DELETE>`
+        :sc-api:`user: delete <User.htm#user_id_DELETE>`
```
Comment (aseemsavio) [inline tenable/sc/users.py:355]: fragment ID should be "user_uuid_DELETE"

**Review by aseemsavio — CHANGES_REQUESTED** [review]
Hi, Thanks for submitting the pull request. Appreciate you spending the time and effort. I've requested some changes. Please feel free to reach me if you have any questions.

Also, please make sure to use the latest branch of master as there are some errors seen in the test cases.

**Comment by lanz** [comment]
> Hi, Thanks for submitting the pull request. Appreciate you spending the time and effort. I've requested some changes. Please feel free to reach me if you have any questions.
> 
> Also, please make sure to use the latest branch of master as there are some errors seen in the test cases.

Thank you for the very quick review @aseemsavio, I've made the requested changes.

**Comment by aseemsavio** [comment]
LGTM

**Review by aseemsavio — APPROVED** [review]

**Review by varap11 — APPROVED** [review]

**Pull request merged**